capstone/src/main/webapp/youtubeScript.js

```diff
@@ -281,3 +275,30 @@ function showCategories() {
     categoryContainer.appendChild(document.createTextNode(" "));
   }
 }
+
+async function callYoutubeServlet(channelId) {
+  const response = await fetch('/youtube_servlet', {
+    method: 'POST',
+    headers: {'Content-Type': 'application/json',},
+    body: channelId});
+  const youtubeServletData = await response.json();
+  return youtubeServletData;
```

Comment:
These two lines, as well as for the two functions below, could be `return await response.json()`

Reply:
addressed in "address nits and add youtube caller support"